README.md

```diff
@@ -1 +1,26 @@
-# atmosphere-integration-tests
\ No newline at end of file
+# atmosphere-integration-tests
+This is the integration tests of the project
+
+## Project setup
+This project depends on:
+>* [atmosphere-agent](https://github.com/MusalaSoft/atmosphere-agent)
+* [atmosphere-client](https://github.com/MusalaSoft/atmosphere-client)
+* [atmosphere-server](https://github.com/MusalaSoft/atmosphere-server)
+* [atmosphere-agent-device](https://github.com/MusalaSoft/atmosphere-agent-device-lib)
+* [atmosphere-commons](https://github.com/MusalaSoft/atmosphere-commons)
+* [atmosphere-client-server](https://github.com/MusalaSoft/atmosphere-client-server-lib)
+* [atmosphere-server-agent](https://github.com/MusalaSoft/atmosphere-server-agent-lib)
+* [atmosphere-ime](https://github.com/MusalaSoft/atmosphere-ime)
+* [atmosphere-service](https://github.com/MusalaSoft/atmosphere-service)
+* [atmosphere-uiautomator-bridge](https://github.com/MusalaSoft/atmosphere-uiautomator-bridge/tree/update-readme)
+
+> Make sure you publish these projects to your local Maven repository
+
+### Chrome driver setup
+* download Chrome Web Driver from [here](https://sites.google.com/a/chromium.org/chromedriver/) and unzip.
+* open `agent.properties` from `atmosphere-integration-tests` root directory and find `chromedriver.executable.path` property. Change the value to point to your chromedriver executable file.
+
+### Build the projects
+Make sure you have connected physical device and emulator.
+You can build the project using the included gradle wrapper by running `./gradlew build` on Linux/macOS
```

Comment:
Probably it's a good idea to separate the commands on different lines so they are more visible.

Also we can add some info about the maven publications.